src/components/OpenSeadragonViewer.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import OpenSeadragon from 'openseadragon';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const styles = {
+  viewer: {
+    display: 'inline-block',
+    width: '100%',
+    height: '700px'
+  }
+};
+
+class OpenSeadragonViewer extends Component {
+  constructor(props) {
+    super(props);
+    this.openSeadragonInstance = null;
+  }
+
+  static propTypes = {
+    itemTitle: PropTypes.string,
+    tileSources: PropTypes.array
+  };
+
+  state = {
+    downloadLink: null
+  };
+
+  componentDidMount() {
+    let { tileSources } = this.props;
+    this.loadOpenSeadragon(tileSources);
+  }
+
+  buildDownloadLink = obj => {
+    // TODO: Figure out a better way to find the event which fires when this is ready
+    setTimeout(() => {
+      let img = this.openSeadragonInstance.drawer.canvas.toDataURL('image/png');
+      this.setState({ downloadLink: img });
+    }, 3000);
+  };
+
+  loadOpenSeadragon(tileSources = []) {
+    const customControlIds = {
+      zoomInButton: 'zoom-in',
+      zoomOutButton: 'zoom-out',
+      homeButton: 'home',
+      fullPageButton: 'full-page',
+      nextButton: 'next',
+      previousButton: 'previous'
+    };
+
+    this.openSeadragonInstance = OpenSeadragon({
+      id: 'openseadragon1',
+      crossOriginPolicy: 'Anonymous',
```

Comment:
I think we need 
```suggestion
      crossOriginPolicy: 'use-credentials',
      loadTilesWithAjax: true,
      ajaxWithCredentials: true,
```
in order to make it work with the authenticating IIIF server.

Reply:
Just pushed up the changes.